app/backend/aquifers/models.py

```diff
@@ -0,0 +1,339 @@
+        verbose_name_plural = 'Aquifer Vulnerability Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class AquiferDemand(AuditModel):
+    """
+    Demand choices for describing Aquifer 
+    -------------------
+    High
+    Low
+    Moderate
+
+    """
+    code = models.CharField(primary_key=True, max_length=1, db_column='aquifer_demand_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'aquifer_demand_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Demand Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class WaterUse(AuditModel):
+    """
+    Type of Known Water Use choices for describing Aquifer 
+    -------------------
+Domestic
+Multiple
+Potential Domestic
+
+    """
+    code = models.CharField(primary_key=True, max_length=2, db_column='water_use_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'water_use_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Water Use Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+class QualityConcern(AuditModel):
+    """
+    -------------------
+Isolated
+Local
+None
+Regional
+    """
+    code = models.CharField(primary_key=True, max_length=2, db_column='quality_concern_code')
+    description = models.CharField(max_length=100)
+    display_order = models.PositiveIntegerField()
+
+    effective_date = models.DateTimeField(blank=True, null=True)
+    expiry_date = models.DateTimeField(blank=True, null=True)
+
+    class Meta:
+        db_table = 'quality_concern_code'
+        ordering = ['display_order', 'code']
+        verbose_name_plural = 'Aquifer Quality Concern Codes'
+
+    def __str__(self):
+        return '%s - %s' % (
+            self.code,
+            self.description
+        )
+
+
+class Aquifer(AuditModel):
+    """
+    An underground layer of water-bearing permeable rock, rock fractures or unconsolidated materials
+    (gravel, sand, or silt), from which groundwater is extracted using a water well. 
+
+    This table holds ONLY the aquifers to which we have associated one or more wells.  It is not
+    the definitive source of all aquifers in the province. 
+
+    """
+    aquifer_id = models.PositiveIntegerField(
+        primary_key=True, editable=False, verbose_name="Aquifer ID Number")
```

Comment:
As discussed, remove editable=False from this and other primary keys

Reply:
done.. note that this was the only usage of 'editable=False" in this file.